py/server/deephaven/time.py

```diff
@@ -877,3 +878,29 @@ def to_np_timedelta64(dt: Union[None, Duration, Period]) -> Optional[numpy.timed
         raise DHError(e) from e
 
 # endregion
+
+# region Utility
+
+def simple_date_format(pattern: str) -> jpy.JType:
+    """ Creates a Java SimpleDateFormat from a date-time format pattern.
+
+    This method is intended for use in Python code when a SimpleDateFormat is needed.
+    It should not be used directly in query strings.
+    The most common use case will use this function to construct a SimpleDateFormat
+    in Python and then use the result in query strings.
+
+    Args:
+        pattern (str): A date-time format pattern string.
+
+    Returns:
+        JObject
+
+    Raises:
+        DHError
+    """
+    try:
+        return _JSimpleDateFormat(pattern)
```

Comment:
@devinrsmith was suggesting a comment here to discourage anyone from ever making a wrapper instead of just returning the Java object.